src/Components/Tabs/FAQ.js

```diff
@@ -0,0 +1,35 @@
+import './FAQ.scss';
+
+import API from '../../Utilities/Api';
+import { Card } from '@patternfly/react-core/dist/esm/components';
+import messages from '../../Messages';
+import React, { useState, useEffect } from 'react';
+import ReactMarkdown from 'react-markdown/with-html';
+import { useIntl } from 'react-intl';
+
+const FAQ = () => {
+    const [content, setContent] = useState();
+    const intl = useIntl();
+
+    useEffect(() => {
```

Comment:
lets comment out the block if it doesn't work yet, no sense in pounding n api for no reason 💯